Sources/MongoSwift/BSON/Document.swift

```diff
@@ -395,8 +417,11 @@ extension Document {
                 if let newValue = newValue {
                     try self.setValue(for: key, to: newValue)
                 } else {
-                    // TODO: SWIFT-224: use va_list variant of bson_copy_to_excluding to improve performance
-                    self = self.filter { $0.key != key }
+                    guard let bson = bson_new() else {
+                        fatalError("Invalid BSON data")
```

Comment:
nit: maybe just use `InvalidBsonError.message`? same below where this is duplicated

Reply:
Done.